### ISSUE TITLE
Can't write to list tags

### ISSUE THREAD

**Issue by Circuit8** [mceachen/exiftool-vendored.js#31]
Hi there,

I can't seem to be able to write to list tags. See https://www.sno.phy.queensu.ca/~phil/exiftool/faq.html#Q17

When I do something along the lines of 

```js
exiftool.write({ SupplementalCategories: [1,2,3] })
```
The resultant metadata is
```xml
<photoshop:SupplementalCategories>
   <rdf:Bag>
    <rdf:li>1,2,3</rdf:li>
   </rdf:Bag>
  </photoshop:SupplementalCategories>
 </rdf:Description>
```
wheras it should be:
```xml
<photoshop:SupplementalCategories>
   <rdf:Bag>
    <rdf:li>1</rdf:li>
    <rdf:li>2</rdf:li>
    <rdf:li>3</rdf:li>
   </rdf:Bag>
  </photoshop:SupplementalCategories>
 </rdf:Description>
```
Exiftool automatically concatenates the list version from above and thus reports on both the same as follows:
```
Supplemental Categories  :  1,2,3
```

Thanks for your help.

**Comment by Circuit8** [comment]
I've had a crack at solving this, the PR is below. I've never used Typescript before so I apologise if I'm making some silly mistakes due to that!

https://github.com/mceachen/exiftool-vendored.js/pull/32

**Comment by mceachen** [comment]
This is fixed by your PR, and is released as of now in version 6.2.0.

Thanks for your work!
